Add MSC2604: Parameters for Login Fallback to the spec. (#2709)
matrix-org/matrix-doc: specification/client_server_api.rst

process. The page will attempt to call the JavaScript function
``window.onLogin`` when login has been successfully completed.

Non-credential related parameters of the login endpoint can be provided as query

> parameters, for example the ``device_id`` or ``initial_device_display_name``.

Author: I would have liked to not list these out so we don't need to potentially update them in the future, but it seemed clearer to be explicit.

> Non-credential related parameters of the login endpoint can be provided as query
> parameters, for example the ``device_id`` or ``initial_device_display_name``.

Reviewer: ```suggestion
Non-credential parameters valid for the ``/login`` endpoint can be provided as query
string parameters here. For example::
```

Reviewer: bonus points if `/login` becomes a link.

Reviewer: oh, rationale: we're not going to remember to update this so it's better to just leave it vague and open.

Author: Right. I couldn't figure out how to make it a link anyway. 😢

Author: Oh hmm, something got funky with accepting this one. Did you expect the "These are to be forwarded to the login endpoint during the login process." line to go away?

Author: I ended up removing that line as well, I think that's what you were going for!

Reviewer: hmm, that line should have survived, I just forgot about it when writing the suggestion - sorry! I'll stuff it back in.
